device/globals/esc_compile.gd

```diff
@@ -205,15 +207,18 @@ func parse_flags(p_flags, flags_list, if_true, if_false, if_inv, if_not_inv):
 			list.push_back(true)
 			flag = flag.substr(1, flag.length()-1)
 			if flag.find("inv-") == -1:
-				if_false.push_back(trim(flag))
+				ifs["if_false"].push_back(trim(flag))
 			else:
-				if_not_inv.push_back(trim(flag).substr(4, flag.length()-1))
+				ifs["if_not_inv"].push_back(trim(flag).substr(4, flag.length()-1))
+		if flag.substr(0, 3) in ["eq ", "ne ", "gt ", "ge ", "lt ", "le "]:
```

Comment:
This should be an `elif`; otherwise negated conditions will be evaluated by the `else` block.

Reply:
Another thing: Should it be possible to negate these new expressions? Possibly, and if so I should probably create a helper function for populating the lists.